partials/subscribe-form.hbs

```diff
@@ -8,57 +8,26 @@
       <div class="sc-bdVaJa iIDDUy">
         <div>
           <div>
-            <p size="16px" color="#434d5d" class="sc-eNQAEJ fBOZaM">Building on
-              video? Subscribe to our newsletter for all the latest product
-              releases, tutorials, and more from Daily.
+            <p size="16px" color="#434d5d" class="sc-eNQAEJ fBOZaM">
```

Comment:
Is `size` and `color` a thing in Handlebars? And what about all these hard-coded class names that look generated?